src/main/java/uk/gov/hmcts/reform/civil/controllers/cases/CasesController.java

```diff
@@ -256,4 +260,14 @@ public ResponseEntity<List<String>> getUserInfo(
         return ResponseEntity.ok(userInformationService.getUserCaseRoles(caseId, authorization));
     }
 
+    @PostMapping(path = "/{caseId}/courtDecision")
+    @Operation(summary = "Get decision on proposed claimant repayment")
+    public RepaymentDecisionType getDecisionOnClaimantProposedRepayment(
+        @PathVariable("caseId") Long caseId,
+        @RequestHeader(HttpHeaders.AUTHORIZATION) String authorisation,
```

Comment:
Follow consistent naming conventions for variables. For example, you can use authorization instead of authorisation for the header parameter.

Reply:
if you have a look further in the controller we use authorisation here too. i prefer the british way of spelling it 
`  public Boolean caseworkerSearchCase(
        @PathVariable("userId") String userId,
        @RequestHeader(HttpHeaders.AUTHORIZATION) String authorization,
        @RequestParam(name = "sdtRequestId") String searchParam
    ) {
        CaseWorkerSearchCaseParams params = CaseWorkerSearchCaseParams.builder()
            .authorisation(authorization)
            .userId(userId)
            .searchCriteria(Map.of("case.sdtRequestIdFromSdt", searchParam)).build();
        List<CaseDetails> caseDetails = caseSdtRequestSearchService.searchCaseForSdtRequest(params);

        if (caseDetails.size() < 1 && caseDetails.isEmpty()) {
            return true;
        }
        return false;
    }
`